src/esp/assets/PTexMeshData.cpp

```diff
@@ -35,12 +36,22 @@ namespace assets {
 
 void PTexMeshData::load(const std::string& meshFile,
                         const std::string& atlasFolder) {
-  ASSERT(io::exists(meshFile));
-  ASSERT(io::exists(atlasFolder));
+  if (!io::exists(meshFile)) {
+    Cr::Utility::Fatal{-1} << "PTexMeshData::load: Mesh file " << meshFile
+                           << " does not exist.";
+  }
+  if (!io::exists(atlasFolder)) {
+    Cr::Utility::Fatal{-1} << "PTexMeshData::load: The atlasFolder "
```

Comment:
@mosra : I used -1 as the exit code. Hope it is fine.

Reply:
Default if you don't specify anything is `1`. Anything except zero is fine I guess, and since we don't document return codes anywhere, the value probably doesn't matter at all.